libklog/libklog.c

```diff
@@ -112,6 +126,18 @@ static void pumpthr(void *arg)
 }
 
 
+void libklog_enable(int enable)
+{
+	if (enable == 0) {
+		libklog_common.enabled = 0;
+	}
+	else {
+		libklog_common.enabled = 1;
+		condSignal(libklog_common.cond);
```

Comment:
Shouldn't this be `condBroadcast`, since multiple threads may now be waiting on the condition?

Reply:
Then number of threads did not change in this PR. Still only the `pumpthr` is listening and there is only one copy of it in each process.